docs/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos.md

```diff
@@ -38,7 +39,7 @@ HOMEBREW_NO_ENV_FILTERING=1 ACCEPT_EULA=Y brew install msodbcsql18 mssql-tools18
 
 The following sections provide instructions for installing previous versions of the Microsoft ODBC driver on macOS.
 
-## <a id="17"></a> Microsoft ODBC 17
+### <a id="17"></a> Microsoft ODBC 17
```

Comment:
Revert. We want the two previous versions to be available via the right-side context links.